packages/react-components/src/components/styles/TogetherMode.styles.ts

```diff
@@ -0,0 +1,186 @@
+ */
+export interface TogetherModeParticipantSeatPosition {
+  height: string;
+  width: string;
+  left: string;
+  top: string;
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Interface for defining the style of a seat position in Together Mode.
+ */
+export interface TogetherModeSeatStyle {
+  seatPosition: TogetherModeParticipantSeatPosition;
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Sets the seating position for a participant in Together Mode.
+ *
+ * @param seatingPosition - The seating position information.
+ * @returns The style object for the seating position.
+ */
+export function setParticipantSeatingPosition(
+  seatingPosition: VideoGalleryTogetherModeSeatingInfo
+): TogetherModeParticipantSeatPosition {
+  return {
+    width: _pxToRem(seatingPosition.width),
+    height: _pxToRem(seatingPosition.height),
+    left: _pxToRem(seatingPosition.left),
+    top: _pxToRem(seatingPosition.top)
+  };
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Return a style bucket based on the number of active sprites.
+ * For example, the first three reactions should appear at maximum
+ * height, width, and opacity.
+ * @private
+ */
+export function setTogetherModeSeatPositionStyle(
+  seatingPosition: VideoGalleryTogetherModeSeatingInfo
+): TogetherModeSeatStyle {
+  return {
+    seatPosition: setParticipantSeatingPosition(seatingPosition)
+  };
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Generates the overlay style for a participant in Together Mode.
+ *
+ * @param seatingPosition - The seating position information.
+ * @returns The style object for the participant overlay.
+ */
+export function getTogetherModeParticipantOverlayStyle(
+  seatingPositionStyle: TogetherModeSeatStyle
+): React.CSSProperties {
+  return {
+    ...seatingPositionStyle.seatPosition,
+    position: 'absolute'
+  };
+}
+
+// Function to map a value from one range to another
+const mapRange = (value: number, inMin: number, inMax: number, outMin: number, outMax: number): number => {
+  return outMin + ((value - inMin) * (outMax - outMin)) / (inMax - inMin);
+};
+
+/**
+ * Calculate the scaled size based on width and height.
+ *
+ * @param width - The width of the element.
+ * @param height - The height of the element.
+ * @returns The scaled size.
+ */
+export const calculateScaledSize = (width: number, height: number): number => {
+  const maxSize = 600;
+  const minSize = 200;
+  const minScaledSize = 35;
+  const maxScaledSize = 70;
+
+  // Use width or height to determine scaling factor
+  const size = Math.min(width, height);
+
+  // Map the size to the desired range
+  return mapRange(size, minSize, maxSize, minScaledSize, maxScaledSize);
+};
+
+/**
+ * Get the root participant status div style.
+ *
+ * @returns The style object for the root participant status div.
+ */
+export const getRootParticipantStatusDivStatus = (): React.CSSProperties => {
```

Comment:
since this is returning a static set of styles every time it can just be the type of `IStyle` from Fluent and you should be able to use it with a fluent component. if this is a div (for overlay reasons) we want to remove the callback option here and make sure we are just setting the value as a constant